test/integration/api/backend_api_configs_controller_test.rb

```diff
@@ -17,10 +17,10 @@ class Api::BackendApiConfigsControllerTest < ActionDispatch::IntegrationTest
   attr_reader :service, :backend_api
 
   test '#index' do
-    service.backend_api_configs.create(backend_api: backend_api)
+    service.backend_api_configs.create!(backend_api: backend_api, path: 'whatever')
     get admin_service_backend_api_configs_path(service)
     assert_response :success
-    assert_select 'table#backend_api_configs tbody tr', count: 2
+    assert_select 'table#backend_api_configs tbody tr', count: service.backend_apis.count
```

Comment:
We should expect a fixed number of backend api configs listed here.

Reply:
@guicassolato I don't understand why :thinking: The goal is that it should show as many as backend apis we have for that service, no? independently of how many there are :thinking: 

But in that case i would suggest adding as the first line of the test: `service.backend_apis.destroy_all`

Reply:
:+1: I think the `service.backend_apis.count` is more robust to any changes

Reply:
I agree with @hallelujah :+1:

Reply:
I'm sorry, but this is incorrect. If the page shows more than 2 backend api configs, then we have a bug. (Specially if it's only for one DB.) Changing this is only hiding that bug.

Reply:
For anybody reading this: Gui, Hery and I are talking about it outside here right now :smile: